about.html

```diff
@@ -0,0 +1,179 @@
+                    id able to ceate content, share content, process data and information
+                    for world wide consmption. Breaking it down, we design websites and mobile
+                    applcation. We create commerscial adverts with high tech resolution.
+                    we market and brand goods sold by creators on our aplication.
+                </p>
+            </div>
+            <div class="headLoc">
+                <p class="headLocw">Please contct us on our email for further questions
+                    about MY Plug <br>
+                    myplug@admin.com
+                </p>
+            </div>
+        </header>
+    </section>
+        <main>
+            <div class="plugDetail" id="valu">
+                <div>
+                    <p class="plgDetPara">
+                        My Plug is an estblished <br>
+                        and creative compnay
+                    </p>
+                    <hr class="plugDetHr">
+                </div>
+                <div>
+                    <p class="valuePara">We value consistance and quality work, that is why
+                        we make sure our clints are satisified with our services
+                    </p>
+                </div>
+                <div class="plugEst">
+                    <div>
+                        <img class="olugI" src="img/plug_icon.png" alt="plug_store">
+                    </div>
+                    <div>
+                    <h4 class="plugEsth">MY PLUG <br>
+                    CREATIVE PLUG</h4>
+                    <p class="plugEsDat">21st - Jan -2022</p>
+                    </div>
+                </div>
+            </div>
+            <div class="project" id="pro">
+                <div>
+                    <h4 class="projectH">Projets that have been completed</h4>
+                    <hr class="projectHr">
+                </div>
+                <div>
+                    <p class="projectPara">
+                        Take a look at the last two projects we have completed.
+                        We put a smile on our clients faces.
+                    </p>
+                </div>
+                <div class="projectDet">
+                    <div class="project1">
+                        <div class="projectBack">
+                            <h4>Coin Mobile applcation</h4>
+                            <p>Mobile application used for cash transaction</p>
+                        </div>
+                    </div>
+                    <div class="project2">
+                        <div class="projectBack">
+                            <h4>Faiz cafe Product Photography</h4>
+                            <p>
+                                Product Photography brands and markets the creators product
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </main>
```

Comment:
**Changes Suggested**
- [ ] The HTML5 `main` element has a higher precedence and should be the outermost for the main body of the webpage except for `navigation elements` Let us replace it with HTML5 `section` element.

Reply:
Fixed